tests/e2e/hcp_machine_pool_test.go

```diff
@@ -1028,6 +1100,18 @@ var _ = Describe("HCP MachinePool", ci.Day2, ci.FeatureMachinepool, func() {
 				mpService.Destroy()
 			}
 		})
+
+		It("worker disk size field - [id:76345]", ci.Low, func() {
+			By("Try to create a nodepool with invalid worker disk size")
+			mpName := helper.GenerateRandomName("np-76345", 2)
+			validateMPArgAgainstErrorSubstrings(mpName, func(args *exec.MachinePoolArgs) {
+				args.DiskSize = helper.IntPointer(10)
+			}, "Must be between 75 GiB and 16384 GiB")
+			validateMPArgAgainstErrorSubstrings(mpName, func(args *exec.MachinePoolArgs) {
+				args.DiskSize = helper.IntPointer(20000)
+			}, "Must be between 75 GiB and 16384 GiB")
+			// TODO terraform plan doesn't have validation
```

Comment:
What this comment is for?

Reply:
For now it uses `tf apply` instead of `tf plan` because `plan` does not have any validations
There is an issue to improve that on provider.
@86254860 Could you improve the comment here and mention the card ?
That way, it will be easier to find out where to update once that is implemented